cli_tools/diagnostics/main_windows.go

```diff
@@ -237,7 +276,7 @@ func gatherLogs(trace bool) ([]logFolder, error) {
 			break
 		}
 	}
-
+	// Note: errors are swallowed if error count <= gathterxxxLogs func count.
```

Comment:
I'm a little wary about this but I'm not reading the code above in detail. If you think this behavior could be improved then please either add a TODO here or file an issue for this repository, describing what would be better to do instead.